addons/storyshots/storyshots-puppeteer/src/imageSnapshot.ts

```diff
@@ -34,10 +35,10 @@ export const imageSnapshot = (customConfig = {}) => {
     getCustomBrowser,
   } = { ...defaultConfig, ...customConfig };
 
-  let browser; // holds ref to browser. (ie. Chrome)
-  let page; // Hold ref to the page to screenshot.
+  let browser: Browser; // holds ref to browser. (ie. Chrome)
+  let page: Page; // Hold ref to the page to screenshot.
 
-  const testFn = async ({ context }) => {
+  const testFn = async ({ context }: any) => {
     const { kind, framework, name } = context;
     if (framework === 'rn') {
```

Comment:
to rename according to https://github.com/storybookjs/storybook/pull/7674/files#diff-0b05142d970443de5c41e81ad14df803R7?

Reply:
Moreover it looks like it match https://github.com/storybookjs/storybook/pull/7674/files#diff-d72af9277db9c8a6cb1b3595750de8c4R9 which is not currently true